Dockerfile

```diff
@@ -10,16 +10,16 @@ MAINTAINER KBase Developer
 
 # -----------------------------------------
 
+RUN pip install --upgrade pip && \
+	pip install pyyaml && \
+	pip install requests
```

Comment:
```suggestion
RUN pip install --upgrade pip && \
	pip install pyyaml == 6.0.1 && \
	pip install requests == 2.32.2
```

Reply:
Fixed, but the python in sdkbase2 is old enough it can't use the most recent versions of requests

Reply:
👍